e2e-tests/gef/cypress/e2e/facility/unissued-facilities/unissued-facilities-change-all-AIN.spec.js

```diff
@@ -409,6 +409,40 @@ context('Unissued Facilities AIN - change all to issued from unissued table', ()
       }
     });
 
+    if (facilityEndDateEnabled) {
+      it('cannot submit facility without bank review date', () => {
+        applicationPreview.submitButtonPostApproval();
+
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateValue().contains('Yes');
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateAction().click();
+
+        aboutFacilityUnissued.isUsingFacilityEndDateNo().click();
+        cy.clickContinueButton();
+
+        cy.visit(relative(`/gef/application-details/${dealId}`));
+
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateValue().contains('No');
+        applicationPreview.facilitySummaryListTable(3).bankReviewDateValue().contains('Required');
+
+        applicationPreview.submitButtonPostApproval().should('not.exist');
+      });
+
+      it('cannot submit facility without facility end date', () => {
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateValue().contains('No');
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateAction().click();
+
+        aboutFacilityUnissued.isUsingFacilityEndDateYes().click();
+        cy.clickContinueButton();
+
+        cy.visit(relative(`/gef/application-details/${dealId}`));
+
+        applicationPreview.facilitySummaryListTable(3).isUsingFacilityEndDateValue().contains('Yes');
+        applicationPreview.facilitySummaryListTable(3).facilityEndDateValue().contains('Required');
+
+        applicationPreview.submitButtonPostApproval().should('not.exist');
+      });
+    }
+
```

Comment:
I think we should split these up so the assertions are more specific/clearer, would help for future debugging as well. What do you think?

```js
it('allows a facility with a bank review date to be submitted', () => { }

it('does not allow a facility without a bank review date to be submitted', () => { }

it('allows a facility with a facility end date to be submitted', () => { }

it('does not allow a facility without a facility end date to be submitted', () => { }
```